linera-execution/tests/test_execution.rs

```diff
@@ -1333,22 +1333,23 @@ async fn test_multiple_messages_from_different_applications() -> anyhow::Result<
 
     // The registration message for the first destination chain
```

Comment:
```suggestion
    // The registration message for the second destination chain
```

Reply:
Or we could rename `first_message` and `second_message` to `message0` and `message1` and the destination chains similarly. (But _keep_ the `*_registration_message`s.)

Reply:
Right, sorry for that. I thought it's about the order in which these variables appear in the code - not about the destination chain.